tasks/base.py

```diff
@@ -291,10 +291,9 @@ def run(self, *args, **kwargs):
                 self._rollback_django()
                 self.retry()
             finally:
-                log_set_task_name(None)
-                log_set_task_id(None)
                 self.wrap_up_dbsession(db_session)
                 self._commit_django()
+                set_log_context(LogContext())
```

Comment:
note: i delayed clearing the log context because `wrap_up_dbsession` and `_commit_django` both have logs in them. may as well keep the task name, etc